### PR TITLE
[WIP]add servicemesh tests on hypershift

### PR DESCRIPTION
**Pull request by shakyav**

### CODE REVIEW

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

**Comment by yxun** [comment]
I am not sure about the config part. Did you run `make update` in your local repo and auto-generating related job yamls ? That may help resolve ci/prow/ check issues

**Comment by yxun** [comment]
Your PR only has ci-operator/config/ file changes now. You may need to run `make update` and auto-generating ci-operator/jobs/ file changes.
The above ci/prow/.... check failed when they compare the ci-operator/jobs/ files

File: ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-main__servicemesh-hypershift-4.14-lp-rosa-hypershift.yaml [hunk 1]
```diff
@@ -55,21 +59,41 @@ tests:
...
       REGION: us-west-2
+      SMCP_VERSION: "2.4"
+      TEST_GROUP: interop
     test:
     - ref: install-operators
     - ref: cucushift-hypershift-extended-health-check
+    - ref: servicemesh-maistra-test-tool-execute
+    - ref: servicemesh-smcp-install
+    - ref: servicemesh-kiali-int-tests-execute
     workflow: firewatch-rosa-aws-sts-hypershift
```
Comment (yxun) [inline ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-main__servicemesh-hypershift-4.14-lp-rosa-hypershift.yaml:97]: is that `firewatch-rosa-aws-sts-hypershift` workflow implemented by another team ?
where is your cluster provisioning step ? or are you login to an existing cluster ?

File: ci-operator/step-registry/servicemesh/kiali/cypress-tests/execute/servicemesh-kiali-cypress-tests-execute-commands.sh [hunk 2]
```diff
@@ -1,18 +1,38 @@
...
-OCP_CRED_PSW="$(cat ${SHARED_DIR}/kubeadmin-password)"
+export OCP_API_URL
+
+# login for interop
+if test -f ${SHARED_DIR}/kubeadmin-password
+then
+  OCP_CRED_USR="kubeadmin"
+  OCP_CRED_PSW="$(cat ${SHARED_DIR}/kubeadmin-password)"
+  oc login ${OCP_API_URL} --username=${OCP_CRED_USR} --password=${OCP_CRED_PSW} --insecure-skip-tls-verify=true
+<<<<<<< HEAD
```
Comment (yxun) [inline ci-operator/step-registry/servicemesh/kiali/cypress-tests/execute/servicemesh-kiali-cypress-tests-execute-commands.sh]: This is a merge conflict. You need to resolve git merge conflicts

Comment (shakyav) [reply ci-operator/step-registry/servicemesh/kiali/cypress-tests/execute/servicemesh-kiali-cypress-tests-execute-commands.sh]: I am reabsing it

File: ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-main__servicemesh-hypershift-4.14-lp-rosa-hypershift.yaml (hunk 1 above)
Comment (shakyav) [reply ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-main__servicemesh-hypershift-4.14-lp-rosa-hypershift.yaml:97]: yes firewatch is added for INTEROP testing

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

**Comment by shakyav** [comment]
closing this, pulling latest changes and to only add hypershift-config

**Pull request closed without merging**